agenta-web/src/components/EvaluationTable/ABTestingEvaluationTable.tsx

```diff
@@ -340,7 +379,22 @@ const ABTestingEvaluationTable: React.FC<EvaluationTableProps> = ({
 
     return (
         <div>
-            <Title className={classes.title}>A/B Testing Evaluation</Title>
+            <Title level={2}>A/B Testing Evaluation</Title>
+            <div>
+                <Row align="middle">
+                    <Col span={12}>
+                        <Button
```

Comment:
A suggestion:

Create a separate component called "SecondaryButton" as a wrapper around this button with its styles. This way we make it reusable, as there's a chance we may use this styled button in other places as well.

If you do this, please don't forget to pass on the rest props to the Button component so that all other props can be passed to it just like a regular antd button.

Something like:

```javascript
(props)=> {
   ....
   return <Button {...props} className={`${classes.button} ${props.className}`} >
}
```